kolibri/core/assets/src/api-resources/channel.js

```diff
@@ -10,4 +10,7 @@ import { Resource } from 'kolibri.lib.apiResource';
 export default new Resource({
   name: 'channel',
   useContentCacheKey: true,
+  fetchFilterOptions(id) {
+    return this.getListEndpoint('filter_options', { id });
```

Comment:
I don't understand why this is needed here.

Reply:
This is just a helper method that's to be able to do call `ChannelResource.fetchFilterOptions(this.topic.channel_id)` we can always do this call directly, but I saw that this is a pattern used in the [ContentResource](https://github.com/endlessm/kolibri/blob/release-v0.14.x/kolibri/core/assets/src/api-resources/contentNode.js#L8), so I implemented in the same way.

Reply:
Ah excellent. It was suspicious because it's the first one in the channel resource.